src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,261 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+
+    private static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE_SEPARATOR = "------------------------------------";
+    private static final String HELLO_MESSAGE = "Hello! I'm duke.Duke\nWhat can I do for you?";
+    private static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+    private static final String ERROR_MESSAGE = "You need to specify the task type!";
+
+    private static final String FILE_PATH = "data/duke.txt";
+
+    private static final String GAP = " / ";
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        loadTasks();
+        printHelloMessage();
+        handleCommand();
+        printByeMessage();
+        saveTasks();
+    }
+
+    private static void loadTasks() {
+        try {
+            addTasksIntoList();
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private static File loadFile() throws IOException {
+        File file = new File(FILE_PATH);
+        if (!file.exists()) {
+            File dir = new File("data");
+            dir.mkdir();
+            File newFile = new File("data/duke.txt");
+            newFile.createNewFile();
+            return newFile;
+        }
+        return file;
+    }
+
+    private static void addTasksIntoList() throws IOException {
+        File f = loadFile();
```

Comment:
Please consider use name like "file" for the File object instead of "f". Similar problems can be found from other variables as well.